Applications/TextEditor/Makefile

```diff
@@ -1,7 +1,8 @@
 include ../../Makefile.common
 
 OBJS = \
-    main.o
+    TextEditor.o \
```

Comment:
I guess this should be named `TextEditorWidget.o` (and .cpp and .h) if it contains the class `TextEditorWidget`.